### PR TITLE
Contac us page

### PR DESCRIPTION
**Pull request by RaunakSharma002**
### Functionality:
created Contact us page for subscription

### Solution:
create contact us page

## WtId:
Dev597

### Risk level:
- [ ] high 
- [x] medium
- [ ] low

### How to test:
by going into My Organisation and click on Contact us button

### CODE REVIEW

File: src/app/body/contact-us/contact-us.component.ts [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+      const personName = this.personName.value;
+      const emailAddress = this.emailAddress.value;
+      const contactNumber = this.contactNumber.value;
+      const orgName = this.orgName.value;
+      const date = this.toolService.date();
+      const time = this.toolService.time();
+  
+      const callable = this.functions.httpsCallable('requestDemo/addRequest');
+      callable({PersonName: personName, EmailAddress: emailAddress, ContactNumber: contactNumber, OrgName: orgName, CreationDate: date, CreationTime: time}).subscribe({
+        next: (data) => {
```
Comment (sanjaykrishna1203) [inline src/app/body/contact-us/contact-us.component.ts:64]: Should add an additional field to differentiate `Request a Demo` and `Contact Us` page.

File: src/app/body/contact-us/contact-us.component.ts [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+          this.enableLoader=false;
+          this.showClose=true;
+        }
+      });
+    }
+    close() {
+      jQuery('#contactUs').modal('hide');
+      this.showClose=false;
+      this.contactUsForm.reset();
+    }
```
Comment (sanjaykrishna1203) [inline src/app/body/contact-us/contact-us.component.ts:83]: We don't use jQuery in our application and the declaration here is also wrong.

Comment (sanjaykrishna1203) [reply src/app/body/contact-us/contact-us.component.ts:83]: just make `this.popupHandlerService.contactUsEnabled` as false that will close the popup

**Review by sanjaykrishna1203 — CHANGES_REQUESTED** [review]
Please resolve merge conflicts

File: src/app/body/contact-us/contact-us.component.html [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+***********************************************************/ -->
+<div class="container-fluid" *ngIf="popupHandlerService.contactUsEnabled">
+    <div class="modal fade" id="contactUs" tabindex="-1" role="dialog" aria-labelledby="contactUsTitle" aria-hidden="true">
+        <div class="modal-dialog modal-dialog-centered" role="document">
+            <div class="modal-content p-2">
+                <div class="modal-body">
+                    <div class="row">
+                        <div class="col-md-10 col-10">
+                            <h2>Request a Demo</h2>
+                        </div>
```
Comment (sanjaykrishna1203) [inline src/app/body/contact-us/contact-us.component.html:23]: Change the title of the modal

**Review by sanjaykrishna1203 — CHANGES_REQUESTED** [review]

File: src/app/body/contact-us/contact-us.component.ts (hunk 1 above)
Comment (RaunakSharma002) [reply src/app/body/contact-us/contact-us.component.ts]: added

File: src/app/body/contact-us/contact-us.component.ts (hunk 2 above)
Comment (RaunakSharma002) [reply src/app/body/contact-us/contact-us.component.ts]: done

File: src/app/body/contact-us/contact-us.component.html (hunk 3 above)
Comment (RaunakSharma002) [reply src/app/body/contact-us/contact-us.component.html]: done

**Pull request closed without merging**